src/content/packer/product-landing.json

```diff
@@ -1,85 +1,103 @@
 {
 	"hero": {
-		"heading": "Packer Placeholder Duplicate Of Vault Content",
+		"heading": "Build Automated Machine Images",
```

Comment:
Not your responsibility to fix since we use it on packer.io, but "Build Automated Machine Images" is a strange tagline ...

What _are_ "automated machine images"? The phrasing could be better, feels handwavy.

I know "Automate Machine Image Builds" is probably confusing or wrong for a different reason that might be worse, but it's technically more correct to what Packer does.

Reply:
Hmmm... Good point. I went with this because it's what's currently on Packer.io, but maybe we can find something better to say here... How does "Automatically Build Machine Images" sound? I think I have to clear this one with marketing so we'll fix it in a different PR, but good catch!